Lib/test/test_email/test_email.py

```diff
@@ -2432,6 +2432,12 @@ def test_multiline_header(self):
         self.assertEqual(str(make_header(decode_header(s))),
                          '"Müller T" <T.Mueller@xxx.com>')
 
+    def test_unencoded(self):
+        # issue 22833
+        s = 'header without encoded words'
```

Comment:
Should also test how this behaves with non-ASCII characters.

Reply:
Will add shortly, per above.